docs/src/configuration.rst

```diff
@@ -3,12 +3,47 @@ Configuration
 =============
 
 
+Initial setup
+=============
+
+Lancet requires a user-level file to identify your account settings such as
+for Gitlab, Harvest, Sentry and others.
+
+The ``lancet setup`` command runs you through the setup step by step and
+generates the ``.lancet`` file automatically for your depending on the
+data provided, here an example::
+
+    [lancet]
+    sentry_dsn = {sentry_dsn_link}
+
+    [tracker:gitlab]
+    username = {gitlab_username}
+
+    [scm-manager:gitlab]
+    username = {gitlab_username}
+
+    [timer:harvest]
+    username = {harvest_id}
+    user_id = {harvest_user_id}
+
+This file gets stored into your user directory ``~/.lancet``.
+
+
 Where does ``lancet`` read the configuration from?
 =================================================
 
+Create a ``.lancet`` configuration file inside your projects root. The contents should be::
 
-Initial setup
-=============
+    [tracker]
+    project_id = {gitlab_project_id}
+    group_id = {gitlab_group_id}
+    paused_status =
+    active_status = doing
+    review_status = review
+
+    [timer]
+    project_id = {harvest_project_id}
+    task_id = {harvest_task_id}
```

Comment:
This can be greatly simplified by moving the common stuff between projects into the user-level config file.

It might be worth to spend 2 sentences about the locations and inheritance between those files (default-settings.ini < ~/.lancet < project/.lancet)

Reply:
adapted docs accordingly